yarn/src/main/scala/org/apache/spark/deploy/yarn/Client.scala

```diff
@@ -185,6 +185,23 @@ private[spark] class Client(
       case None => logDebug("spark.yarn.maxAppAttempts is not set. " +
           "Cluster's default value will be used.")
     }
+    sparkConf.getOption("spark.yarn.attemptFailuresValidityInterval").map(_.toLong) match {
```

Comment:
You should use `sparkConf.getTimeAsMs` here. Since there's no `Option` equivalent for that one, you'll need to use `sparkConf.contains` to check if the value is set.